app/client/src/modules/ui-builder/ui/wds/WDSComboBoxWidget/widget/index.tsx

```
@@ -1,187 +1,59 @@
-
-  static getPropertyPaneStyleConfig() {
-    return [];
-  }
-
-  static getDerivedPropertiesMap() {
     return {
-      selectedOption:
-        "{{_.find(this.options, { value: this.selectedOptionValue })}}",
-      isValid: `{{ this.isRequired ? !!this.selectedOptionValue : true }}`,
-      value: `{{this.selectedOptionValue}}`,
+      ...super.getConfig(),
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)